src/processing/trigonometri/trigonometri.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Trigonometri, regulære mangekanter og stjerner
+level: 2
+author: Sigmund Hansen
+---
+
+# Introduksjon {intro}
+
+Nå som du kan tegne [mangekanter](../mangekanter/mangekanter.html)
+(hvis du ikke har gjort leksjonen om mangekanter, bør du gjøre dem
+først), skal vi se på en litt spesiell type mangekanter: de regulære
+mangekantene. Det vil si de mangekantene hvor lengden av hver kant er
+lik og vinkelen i hvert hjørne er lik. Vi skal også tegne deres nære
+slektninger, de regulære stjernene.
+
+Det er en stor fordel å kunne litt trigonometri før man slår seg løs
+på disse oppgavene, men vi skal prøve å gi korte forklaringer av de
+konseptene som brukes i leksjonen.
+
+## Sinus og cosinus {.protip}
+
+Før vi begynner å tegne regulære mangekanter, skal vi bare ta en liten
+titt på to trigonometriske funksjoner som vi kommer til å bruke mye:
+sinus og cosinus. Vi skal se på dem spesifikt i forbindelse med
+sirkler.
+
++ Ethvert punkt langs omrisset av sirkelen befinner seg like langt fra
+  midten av sirkelen. Denne avstanden er radius i sirkelen, som regel
+  skriver vi bare `r` i figurer og formler.
+
+![](SirkelRadius.png "En sirkel med linjer med lengde `r` fra sentrum av sirkelen til punkter i omrisset")
+
++ Vi kan tegne en rettvinklet trekant som ligger vannrett og strekker
+  seg fra midten av sirkelen til ett av disse punktene.
+
+![](RettvinkletTrekantSirkel.png "En rettvinklet trekant mellom et punkt i sirkelens omriss og sentrum.")
+
++ Hvis vi sier at sentrum av sirkelen ligger i punktet (0, 0), altså X
+  og Y er null i midten av sirkelen, kan vi enkelt regne ut X og Y for
+  punktet i omrisset. To av sidene i trekanten ovenfor viser da X og
+  Y. Den siste siden er linjen fra sentrum med lengde `r`. Her har vi
+  kalt sidene `a`, `b` og `c`; navnet kan brukes for lengdene til

```

Comment:
Ja, det går an å kutte ut a, b, c. Det er vel hovedsakelig en vanesak at a, b og c brukes for å benevne sidene i en trekant. A, B og C for hjørnene og alfa, beta og gamma for vinklene.